ballerina/modules/oas/client.bal

```diff
@@ -0,0 +1,4908 @@
+    # + updated_at_min - Show smart collections last updated after this date. (format: 2014-04-25T16:15:47-04:00) 
+    # + updated_at_max - Show smart collections last updated before this date. (format: 2014-04-25T16:15:47-04:00) 
+    # + published_at_min - Show smart collections published after this date. (format: 2014-04-25T16:15:47-04:00) 
+    # + published_at_max - Show smart collections published before this date. (format: 2014-04-25T16:15:47-04:00) 
+    # + published_status - Filter results based on the published status of smart collections.(default: any) 
+    # + fields - Show only certain fields, specified by a comma-separated list of field names. 
+    # + xShopifyAccessToken - The access token for the shop
+    # + return - Retrieve a list of all smart collections for a certain product_id / Retrieve a list of specific smart collections / Retrieve a list of all smart collections / Retrieve a list all smart collections after a specified ID / Retrieve a list of all smart collections for a certain product_id / Retrieve a list of specific smart collections / Retrieve a list of all smart collections / Retrieve a list all smart collections after a specified ID 
+    remote isolated function retrievesAListOfSmartCollections(string apiVersion, string? 'limit = (), string? ids = (), string? since_id = (), string? title = (), string? product_id = (), string? 'handle = (), string? updated_at_min = (), string? updated_at_max = (), string? published_at_min = (), string? published_at_max = (), string? published_status = (), string? fields = (), string? xShopifyAccessToken = ()) returns SmartCollectionList|error {
+        string resourcePath = string `/admin/api/${getEncodedUri(apiVersion)}/smart_collections.json`;
+        map<anydata> queryParam = {"limit": 'limit, "ids": ids, "since_id": since_id, "title": title, "product_id": product_id, "handle": 'handle, "updated_at_min": updated_at_min, "updated_at_max": updated_at_max, "published_at_min": published_at_min, "published_at_max": published_at_max, "published_status": published_status, "fields": fields};
+        resourcePath = resourcePath + check getPathForQueryParam(queryParam);
+        map<any> headerValues = {"x-shopify-access-token": xShopifyAccessToken};
+        map<string|string[]> httpHeaders = getMapForHeaders(headerValues);
+        SmartCollectionList response = check self.clientEp->get(resourcePath, httpHeaders);
+        return response;
+    }
+    # Creates a smart collection
+    #
+    # + apiVersion - The API version
+    # + xShopifyAccessToken - The access token for the shop
+    # + return - Create a new smart collection with a base64 encoded image / Create a smart collection of all products starting with the specified term / Create a smart collection with a specified title / Create a new smart collection with an image that will be downloaded by Shopify / Create a new unpublished smart collection / Create a new smart collection with a base64 enc
```

Comment:
Can't we sanitize the spec to use better names than this `InlineResponse200179`?